src/patternfly/components/Button/examples/button-states-example.hbs

```diff
@@ -76,6 +76,6 @@
 {{#> button button--modifier="pf-m-plain pf-m-active" button--attribute='aria-label="Remove"'}}
   {{> button-icon button-icon--type="times"}}
 {{/button}}
-{{#> button button--modifier="pf-m-plain" button--attribute="disabled" button--attribute='aria-label="Remove"'}}
-  {{> button-icon button-icon--type="times"}}
+{{#> button button--modifier="pf-m-plain" button--attribute='disabled aria-label="Remove"'}}
+  {{> button-icon button-icon--type="times" aria-hidden="true"}}
```

Comment:
The update on line 79 is great! But we don't need to add `aria-hidden="true"` here. It's already included as part of the `button-icon` handlebar.

Reply:
`aria-hidden="true"` has been removed.